modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditAppFooter.es.js

```diff
@@ -168,7 +154,7 @@ export default withRouter(
 							<Button
 								disabled={
 									appDeployments.length === 0 ||
-									!appName ||
+									!appName?.trim() ||
```

Comment:
Could you use `name[editingLanguageId]` instead of `appName`? Because we only use it once.

Reply:
Sure!!